### PR TITLE
Explain when to use fastify in a new FAQ section

### PR DESCRIPTION
**Pull request by binarykitchen**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,14 @@ Everyone is very welcome to contribute to this project.
 You can contribute just by submitting bugs or suggesting improvements by
 [opening an issue](/../../issues) or by [sending a pull request](/../../pulls).
 
+## FAQ
+
+  1. **Why is the public Fastify site not running on Fastify when you are promoting it?**
```
Comment (Eomm) [inline README.md:73]: It is not a "Frequently Asked Questions" - nobody asked for it before. I think the opened issue is enough here

File: README.md [hunk 2]
```diff
@@ -68,6 +68,14 @@ Everyone is very welcome to contribute to this project.
...
+  Well, good question, but think twice: it is serving plain static files only. And we are honest while foundation money is precious. That said, NGINX is good enough, until you have API calls ...
+  2. **When should I not use Fastify?**
+  Take a step back. Know your problems first. Compare these with our advantages + disadavantages.
```
Comment (Eomm) [inline README.md:76]: This does not answer - it is a general programming rule to use the right tool for the right task

File: README.md [hunk 3]
```diff
@@ -68,6 +68,14 @@ Everyone is very welcome to contribute to this project.
...
+  99. **But, I have more questions?**
+  Sure. Just think twice, then submit a [new issue](https://github.com/fastify/website/issues) and formulate well. We will reply :)
```
Comment (Eomm) [inline README.md:78]: In the main repo we have this section:
https://github.com/fastify/fastify#support
and in the website we have this `help` button:

image

**Review by Eomm — CHANGES_REQUESTED** [review]
I don't think this section has a real value tbh

Moreover, this repo is going to be archived in 1 or 2 weeks due: https://github.com/fastify/website-next

**Comment by binarykitchen** [comment]
Good points, and didn't know a new website was coming soon. 
Never mind, let's close, and thanks for your feedback.

**Pull request closed without merging**